python/Pulumi.yaml

```diff
@@ -1,4 +1,4 @@
-name: metal-k8s-cluster
+name: equinix-py-metal-k8s-cluster
```

Comment:
It looks like this PR also includes the changes from #8.

Reply:
Right! I accidentally opened this branch from `python_ccm_and_kubevip_load_balancer` instead of `main`. Perhaps I can just close #8 in favor of this one and rename and update this description